src/components/staking/LiValidator.vue

```diff
@@ -18,73 +18,61 @@
         <template v-slot="{ result: { loading, error, data: keybase } }">
           <Avatar
             v-if="!keybase || !keybase.avatarUrl || loading || error"
-            class="data-table__row__info__image data-table__row__info__image--no-img"
+            class="li-validator-image"
             alt="generic validator logo - generated avatar from address"
             :address="validator.operator_address"
           />
           <img
             v-else-if="keybase && keybase.avatarUrl"
             :src="keybase.avatarUrl"
-            class="data-table__row__info__image"
+            class="li-validator-image"
             :alt="`validator logo for ` + validator.description.moniker"
           />
         </template>
       </ApolloQuery>
-      <div class="data-table__row__info__container">
-        <span
-          v-tooltip.top="status"
-          :class="statusColor"
-          class="data-table__row__info__container__status"
-        />
-        <span class="data-table__row__info__container__name">
-          {{
-          validator.description.moniker
-          }}
-        </span>
-        <div class="data-table__row__info__container__description">
-          <Bech32 :address="validator.operator_address" />
+      <div class="validator-info">
+        <h3 class="li-validator-name">
+          {{ validator.description.moniker }}
+        </h3>
+        <div v-if="validator.my_delegations > 0">
+          <h4>
+            {{
+              validator.my_delegations
```

Comment:
You don't need the ternary operator, since the `v-if` above already checks it's above 0. You could also use filters to make it look nicer.